docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,460 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
+---
+
+## v2.3
+
+### Additions
+
+`@interfaceObject` a directive that allows you to abstract an interface in isolation by declaring it locally as an object type. This enables you to contribute fields without needing to redefine all the interface’s concrete implementation types.
+
+```graphql
+directive @interfaceObject on OBJECT
+```
+
+`@key` on interface definitions, in order to resolve the implementation type of the abstracted objects returned by the subgraphs using `@interfaceObject`.
```

Comment:
technically definition already defined interfaces as a valid location (but it didn't work) -> v2.3 actually fixes that

Reply:
do you think it is worth mentioning anyway, or shall I drop it?

Reply:
there is no change required on the lib side so don't think there is a need for it. I'd call out that when extending interfaces through`@interfaceObject` types, that subgraph have to handle the "interface" __typename in the _entities resolver